lobby-db/src/main/resources/db/migration/V2.00.00__lobby.sql

```diff
@@ -49,7 +49,7 @@ alter table lobby_user
     owner to lobby_user;
 alter table lobby_user
     add constraint lobby_user_pass_check check (password IS NOT NULL OR bcrypt_password IS NOT NULL);
-comment on table lobby_user is 'Stores information about TripleA Lobby users.';
+comment on table lobby_user is 'Stores information about Triplea Lobby users.';
```

Comment:
Not sure about that, but don't we usually use `TripleA` as default way to write it?

Reply:
Yes, but it's also camel-cased too..

Reply:
Not quite sure on how to reply to your statement.
I mean shouldn't we agree on some way or another?

Reply:
I'm not sure i follow what kind of suggestion you would be proposing @RoiEXLab , could you walk me through what you might want to see?

The rule of looking for camel case in SQL files is pretty naive, just any lower case letter next to upper case. I don't think the ROI for special casing 'TripleA' to that is significant enough to worry about, and I would be concerned that we could see examples where 'TripleA' were used in a table or column name (in general, or let alone as a camel cased variant)

Reply:
Ahh, that's the missing piece I was missing: There's a script that checks the casing in this file, which isn't smart enough to differenciate between regular syntax and strings, right?
In this case feel free to merge, I can't think of a good way to enhance the script to ignore strings. I mean for this particular case it might be sufficient to run the file through sed first and replace all strings with the empty string, but this could potentially cause problems with escape sequences